frontend/src/styles/landing.css

```diff
@@ -1,3 +1,37 @@
+/* Place jumbotron in the center of the screen, with slight transparency. */
+.jumbotron {
+  background-color: rgba(255, 255, 255, 0.8);
+  left: 50%;
+  padding: 5%;
+  position: absolute;
+  text-align: center;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+#title {
+  font-size: 10vh;
+  margin: 0;
+}
+
+#subtitle {
+  color: var(--SLURP-GRAY);
+  font-size: 3vh;
+  opacity: 50%;
+}
+
+#landing-description {
+  color: var(--SLURP-BLUE);
+  font-size: 2vh;
```

Comment:
Did you test this on multiple window sizes?

Reply:
Yeah I played around with multiple window sizes and it looks fine.